code/game/objects/items/crayons.dm

```diff
@@ -871,7 +871,7 @@
 			. += "It's roughly [PERCENT(charges_left/charges)]% full."
 		else
 			. += "It is empty."
-	. += span_notice("Alt-click [src] to [ is_capped ? "take the cap off" : "put the cap on"]. Right-click a colored object to match its existing color.")
+	. += span_notice("Alt-click [src] to [ is_capped ? "take the cap off" : "put the cap on"].")
```

Comment:
Why not remove this entirely? Spraycans have tooltips enabled.

Reply:
Not everyone has them on, doesn't hurt

Reply:
Oof, I forgot that was a preference setting.